larq_compute_engine/mlir/transforms/passes.h

```diff
@@ -1,41 +1,47 @@
 #ifndef LARQ_COMPUTE_ENGINE_MLIR_PASSES_H_
 #define LARQ_COMPUTE_ENGINE_MLIR_PASSES_H_
 
+#include "mlir/Dialect/Func/IR/FuncOps.h"
 #include "mlir/Pass/Pass.h"
 
 enum LCETarget { ARM = 0, XCORE = 1 };
 
 namespace mlir {
+// namespace func {
+// class FuncOp;
+// }
```

Comment:
Minor thing, but I guess this can be removed?
```suggestion
```

Reply:
Good catch, that probably should not have been committed in the first place